pkg/db/sqlite/sqlite.go

```diff
@@ -215,6 +215,18 @@ var headerFieldToColumnMap = map[string]string{
 	"value": "value",
 }
 
+func (c *Client) ClearRequestLogs(ctx context.Context) error {
+	tx := c.db.MustBegin()
+	tx.MustExec("DELETE FROM http_headers")
+	tx.MustExec("DELETE FROM http_requests")
```

Comment:
Because we use [foreign key support](https://github.com/dstotijn/hetty/blob/v0.2.1/pkg/db/sqlite/sqlite.go#L84) and have `ON DELETE CASCADE` clauses on the foreign key constraints, it should suffice to `DELETE FROM http_requests` to delete all related rows from the other two tables as well.